shell.nix

```diff
@@ -1,31 +1,21 @@
 # Credit to https://github.com/profunktor/redis4cats/blob/master/shell.nix
 let
-  config = {
-    packageOverrides = pkgs: rec {
-      sbt = pkgs.sbt.overrideAttrs (
-        old: rec {
-          version = "1.3.13";
-
-          patchPhase = ''
-            echo -java-home ${pkgs.openjdk11} >> conf/sbtopts
-          '';
-        }
-      );
-    };
+  jdk11 = self: super: rec {
```

Comment:
Note this overlay will override the `jre` version of every single package that requires it, including `sbt`, but will *not* override `sbt`'s version, which the previous version was doing. Unless something changed upstream in `nixpkgs`, that's how things were last I time I checked.

Reply:
I don't think we need to care about sbt's version honestly. It's the launcher and the pre-fetched version, sure, but `build.properties` defines the sbt version anyway - and is managed by scala steward, and I we were already out of sync with it.

btw. I'm dumb, used `pkgs` instead of `self` :D it was a miracle it worked

Reply:
I usually try to keep them in sync out of habit. E.g., I'm not sure if a newer version in `build.properties` would make `sbtn` available if your binary is an older version which doesn't have it, probably try that if interested.

Reply:
Ha! In that case, either `self`, `super` or `pkgs` would do the same thing :)

Reply:
yeah, that's the miracle

Reply:
So do we believe this is good to merge?

Reply:
Now i do

Reply:
👍 thanks very much!